src/odbc/winsetup.c

```diff
@@ -428,6 +433,7 @@ check_installer_errors(void)
 	return SQL_SUCCEEDED(ret);
 }
 
+#ifdef DLL_EXPORT
```

Comment:
yes, here it makes sense `DLL_EXPORT`, you cannot register something not an ODBC driver.